private/testplanet/satellite.go

```diff
@@ -227,10 +227,11 @@ func (planet *Planet) newSatellites(count int) ([]*SatelliteSystem, error) {
 		if planet.config.Reconfigure.NewSatelliteDB != nil {
 			db, err = planet.config.Reconfigure.NewSatelliteDB(log.Named("db"), i)
 		} else {
+			schema := satellitedbtest.SchemaName(planet.id, "S", i, "")
 			if *pgtest.CrdbConnStr != "" {
-				db, err = satellitedb.New(log, *pgtest.CrdbConnStr)
+				namespace := strings.ReplaceAll(schema, "/", "")
```

Comment:
It looks like this transformation is done every time you call `NewCockroach`. What about moving the `strings.ReplaceAll()` call inside `NewCockroach` instead?

Then if we ever change how we want the name-sanitizing to work, it only has to be done in one place.

Reply:
using `pq.QuoteIdentifier()` I actually don't need the `strings.ReplaceAll()` anymore

Reply:
Oh yeah good point! nice.